test/acceptance/features/examples.feature

```diff
@@ -53,3 +54,37 @@ Feature: Verify examples provided in Service Binding Operator github repository
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "sbr-to-bind-hello-app-to-route" should be changed to "True"
         And jq ".status.conditions[] | select(.type=="Ready").status" of Service Binding "sbr-to-bind-hello-app-to-route" should be changed to "True"
         And Secret "sbr-to-bind-hello-app-to-route" contains "ROUTE_HOST" key with value "example-sbo.apps.ci-ln-smyggvb-d5d6b.origin-ci-int-aws.dev.rhcloud.com"
+
+    # https://github.com/redhat-developer/service-binding-operator/tree/master/examples/nodejs_postgresql_namespaces
+    Scenario: Binding an Imported app in one namespace to an In-cluster Operator Managed PostgreSQL Database in another namespace
```

Comment:
how about to rephrase it to:

```
Bind application to a database running in another namespace
```

Reply:
The scenario name deliberately kept the same as example name. The suggested name sounds cool, should I also update the [example name](https://github.com/redhat-developer/service-binding-operator/blob/master/examples/nodejs_postgresql_namespaces/README.md#binding-an-imported-app-in-one-namespace-to-an-in-cluster-operator-managed-postgresql-database-in-another-namespace)?

Reply:
Yes, please.

Reply:
Done. PTAL.